common/openapi.yml

```diff
@@ -302,6 +303,19 @@ components:
         array_type:
           $ref: ParameterTypeDefinition
 
+    ParameterTypeDefinitionMap:
+      type: object
+      required:
+        - key_type
+        - value_type
+      allOf:
+        - $ref: '#/components/schemas/ParameterTypeDefinition'
+      properties:
+        key_type:
```

Comment:
Should only allow primitive types

Reply:
Is this as easy as changing `$ref: ParameterTypeDefinition` to `$ref: ParameterTypeDefinitionPrimitive`?

Reply:
Yes... almost 😆 . This is the commit, with a few extra fixups: https://github.com/DataBiosphere/cbas/pull/117/commits/e5e12fb1824a4f0c2c86caf68e15c319741d6d54